fossunited/hooks.py

```diff
@@ -84,3 +84,15 @@
     {"source": r"c/(.+)/cfp/all", "target": r"/dashboard/cfp/\1"},
     {"source": r"c/(.+)/schedule", "target": r"/dashboard/schedule/\1"},
 ]
+
+# Installation
+# ----------------
+after_install = "fossunited.install.after_install"
+
+# Uninstallation
+# ----------------
+before_uninstall = "fossunited.uninstall.before_uninstall"
+
+override_doctype_class = {
+    "Newsletter": "fossunited.overrides.newsletter_master.NewsletterMaster",
+}
```

Comment:
q: not sure why we need a different `Newsletter` doctype.

Reply:
The way newsletter naming works in the frappe framework, is that the name = newsletter subject. This will definitely cause problems in our usecase. Here, we are overriding the newsletter doctype and adding a [`autoname`](https://frappeframework.com/docs/user/en/basics/doctypes/controllers) controller to have the name to be a random hash